src/components/Process/View.tsx

```diff
@@ -71,6 +75,39 @@ export const ProcessView = () => {
     if (election?.status === ElectionStatus.RESULTS) setTabIndex(1)
   }, [election])
 
+  useEffect(() => {
+    if (!err) return
+
+    if (err === 1) return setErr((prev) => prev + 1)
```

Comment:
Single line returns are ok for the first case, but the second one would be better in brackets

```suggestion
    if (!err) return

    if (err === 1) {
      return setErr((prev) => prev + 1)
    }
```